docs/development/database_migrations.rst

```diff
@@ -157,3 +157,12 @@ accurately test your migration, and it will likely break during future code chan
 should use as few dependencies as possible in your test including other ``securedrop`` code as well
 as external packages. This may be a rather annoying requirement, but it will make the tests more
 robust aginst future code changes.
+
+Release Testing Migrations
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to ensure that migrations between from the previous to current version of SecureDrop apply
+cleanly in production-like instances, we have a helper script that is designed to load
+semi-randomized data into the database. You will need to modify the script ``qa_loader.py`` to
+include sample data. This sample data should intentionally include edge cases that might behave
+strangely such as data whose nullability is only enforced by the application or missing files.
```

Comment:
For release managers and release testers, let's add a bit of docs saying:

1. Provision staging or prod VMs.
2. `sudo su`
3. `cd /var/www/securedrop`
4. `./qa_loader.py`